core/templates/pages/contributor-dashboard-page/services/contribution-opportunities-backend-api.service.ts

```diff
@@ -215,6 +215,29 @@ export class ContributionOpportunitiesBackendApiService {
       return [];
     }
   }
+
+  async saveUserLanguagePreferenceAsync(language_code: string):
+  Promise<unknown> {
```

Comment:
Should this return a void Promise? 

```suggestion
  Promise<void> {
```

Reply:
I have changed this `<Object>` instead. `<void>` was throwing an error.
![image](https://user-images.githubusercontent.com/75239620/168761541-c94b47c9-9cc3-474b-8c7f-f62cebb2c0a1.png)

Reply:
I think you need to specify the Promise return type. Added a comment.